MLS.Agent.Tests/CommandLine/VerifyCommandTests.cs

```diff
@@ -135,6 +135,41 @@ public async Task Files_are_listed()
                            $"{root}{Path.DirectorySeparatorChar}doc.md*Line 2:*{root}{Path.DirectorySeparatorChar}Program.cs (in project {root}{Path.DirectorySeparatorChar}some.csproj)*".EnforceLF());
             }
 
+            [Fact]
+            public async Task Fails_if_language_is_not_compatible_with_backing_project()
+            {
+                var root = Create.EmptyWorkspace(isRebuildablePackage: true).Directory;
+
+                var directoryAccessor = new InMemoryDirectoryAccessor(root, root)
+                {
+                    ("some.csproj", CsprojContents),
+                    ("Program.cs", CompilingProgramCs),
+                    ("support.fs", "let a = 0"),
+                    ("doc.md", @"
+```fs --source-file support.fs
+```
+")
+                }.CreateFiles();
```

Comment:
Can this test do without creating the files on disk or is the `verify` command still dependent on real project files?

Reply:
it si still looking for real file in verify